data/incipias/incipias outfits.txt

```diff
@@ -0,0 +1,297 @@
+
+# Engines:
+outfit "MH Blue-Class Engine"
+	category "Engines"
+	thumbnail "outfit/mh blue-class engine"
+	"mass" 27
+	"outfit space" -27
+	"engine capacity" -27
+	"flare sprite" "effect/incipias flare/metallic hydrogen small"
+		"frame rate" 24
+	"flare sound" "plasma huge"
+	"thrust" 24
+	"thrusting energy" 2
+	"thrusting heat" 2.5
+	description "In Incipian culture, blue is the weakest color. An engine labeled with blue does not bode well for the speed of any ship using it."
+
+outfit "MH Blue-Class Steering"
+	category "Engines"
+	thumbnail "outfit/mh blue-class steering"
+	"mass" 24
+	"outfit space" -24
+	"engine capacity" -24
+	"steering flare sound" "plasma huge"
+	"turn" 498
+	"turning energy" 1.
+	"turning heat" 3
+	description "This steering barely provides the minimum force needed to turn a ship, but it still used often because of how cheap it is."
+
+outfit "MH Green-Class Engine"
+	category "Engines"
+	thumbnail "outfit/mh green-class engine"
+	"mass" 32
+	"outfit space" -32
+	"engine capacity" -32
+	"flare sprite" "effect/incipias flare/metallic hydrogen medium"
+		"frame rate" 24
+	"flare sound" "plasma huge"
+	"thrust" 30
+	"thrusting energy" 2
+	"thrusting heat" 2.5
+	description "Green is a color of balance for the Incipias. This engine balances the need for space with the need for powerful thrusters."
+
+outfit "MH Green-Class Steering"
+	category "Engines"
+	thumbnail "outfit/mh green-class steering"
+	"mass" 28
+	"outfit space" -28
+	"engine capacity" -28
+	"steering flare sound" "plasma huge"
+	"turn" 618
+	"turning energy" 1.1
+	"turning heat" 3.1
+	description "The Incipias seem to have something against turning their ships. Even though this steering is supposed to be an upgrade from the Blue-Class steering, it is barely an improvement."
+
+outfit "MH Red-Class Engine"
+	category "Engines"
+	thumbnail "outfit/mh red-class engine"
+	"mass" 41
+	"outfit space" -41
+	"engine capacity" -41
+	"flare sprite" "effect/incipias flare/metallic hydrogen large"
+		"frame rate" 24
+	"flare sound" "plasma huge"
+	"thrust" 48
+	"thrusting energy" 5
+	"thrusting heat" 10
+	description "The Incipias burn metallic hydrogen to move their ships, allowing for very efficient engines. The abundance of the fuel source on their home planet is also a benefit."
+
+outfit "MH Red-Class Steering"
+	category "Engines"
+	thumbnail "outfit/mh red-class steering"
+	"steering flare sound" "plasma huge"
+	"mass" 41
+	"outfit space" -41
+	"engine capacity" -41
+	"turn" 1134
+	"turning energy" 2.5
+	"turning heat" 12
+	description "Even the Incipias realize that ships need to turn towards their target to be able to shoot it. This steering is specifically designed for combat but captains with a lot of money find it very useful for mining."
```

Comment:
```suggestion
	description "Even the Incipias realize that ships need to turn towards their target to be able to shoot it. This steering is specifically designed for combat, but captains with a lot of money find it very useful for mining."
```
There haven’t actually been any battles the Incipias have been involved in, right?

Reply:
internal battles did happen although on small scale and more for deterrence